srml/democracy/src/lib.rs

```diff
@@ -189,6 +240,11 @@ decl_module! {
 		}
 
 		/// Undelegate vote.
+		///
+		/// # <weight>
+		/// - Depends on weight of `T::Currency::set_lock()`
```

Comment:
We don't include `()` on our functions.
```suggestion
		/// - Depends on weight of `T::Currency::set_lock`.
```